frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PipelineForm from '../../pipeline-form/PipelineForm';
+import PipelineParameters from '../PipelineParameters';
+
+const pipelineFormProps: React.ComponentProps<typeof PipelineForm> = {
+  PipelineFormComponent: PipelineParameters,
+  formName: 'parameters',
+  obj: {},
+};
+
+describe('Pipeline details parameter tab for pipeline with empty parameters', () => {
+  const obj = { metadata: { name: 'my-pipeline', namespace: 'test' } };
+  const expected = { parameters: [], resources: [] };
+  const plFormWrapper: ShallowWrapper = shallow(<PipelineForm {...pipelineFormProps} obj={obj} />);
+
+  it('Should render a formik component', () => {
+    expect(plFormWrapper.find('Formik').exists()).toBe(true);
+  });
+
+  it('Should render the pipeline parameters form', () => {
+    const formWrapper = plFormWrapper.find('Formik').shallow();
+    expect(formWrapper.find('PipelineParameters').exists()).toBe(true);
+  });
+
+  it('Should render the correct parameters', () => {
+    const formWrapper = plFormWrapper.find('Formik').shallow();
+    const params = formWrapper.find('PipelineParameters').prop('values');
+    expect(params).toEqual(expected);
+  });
+});
+
+describe('Pipeline details parameter tab for pipeline with parameters in place', () => {
```

Comment:
Why two describe blocks? It looks like you're testing the same component but split up the tests for no gain and duplicated some effort.